plugins/jira/help.md

```diff
@@ -1143,6 +1200,7 @@ _This plugin does not contain any troubleshooting information._
 
 # Version History
 
+* 6.1.0 - Add Updated Issue trigger | Update New Issue trigger to only trigger in case of recently created tickets | Change `required` property to false for `project` input in New Issue trigger | Fix issue in New Issue trigger to include all results that match JQL | Fix issue in New Issue trigger with retrieving attachments
```

Comment:
```suggestion
* 6.1.0 - Add Monitor Issues trigger | Update New Issue trigger to only trigger in case of recently created tickets | Change `required` property to false for `project` input in New Issue trigger | Fix issue in New Issue trigger to include all results that match JQL | Fix issue in New Issue trigger with retrieving attachments
```